filters/locale-date.js

```diff
@@ -0,0 +1,29 @@
+(function () {
```

Comment:
I like the filters approach, storing raw data in backend and transforming as needed when it has to be displayed  :+1: 

I gess they're used as translate in view templates.

If we just decouple the function from the angular filter definition, I suppose that this also allow us to reuse the functions code if needed, packaged in a node module or similar, to use the same code to transform in frontend and in backend if some time we need it there also...

Reply:
wouldn't it easier to apply the predefined date filter?

Reply:
Are we needing this in other place? If so, for sure we shall create a package. If not, maybe it's not the moment to do that (?)

Reply:
@DevStarlight Date filter doesn't allow you to set the date in design required format. Or if it does, I didn't found the way. Specially, to use locale strings.

Reply:
No, i think it's not needed by now, just thinking about it :)

And for the date localization, it seems it can be done with an angular module, if we want to explore it for the future: 
https://stackoverflow.com/a/32111231